dashboard/app/models/course_offering.rb

```diff
@@ -385,6 +386,7 @@ def self.properties_from_file(content)
   # seed_all
   def self.seed_record(file_path)
     properties = properties_from_file(File.read(file_path))
+    properties.delete(:self_paced_pl_course_offering_key)
```

Comment:
I would expect the seed code to use the key to look up the self-paced pl course, and populate the self_paced_pl_course_offering_id in the db here.